koku/koku/celery.py

```diff
@@ -116,6 +116,12 @@ def task(self, *args, **kwargs):
     'args': []
 }
 
+# Collect prometheus metrics.
+app.conf.beat_schedule['db_metrics'] = {
+    'task': 'koku.metrics.collect_metrics',
+    'schedule': crontab(hour='*/3'),
```

Comment:
```suggestion
    'schedule': crontab(minute='*/15'),
```

Reply:
Let's run this more frequently. It should be a quick task, and there's issues at the top of the backlog right now to add some error state checking to the metrics, so if we collect more frequently we can alert on errors faster.